pages/my-templates/[templateId].tsx

```diff
@@ -52,6 +52,7 @@ const MyNFTsTemplateDetail = (): JSX.Element => {
   const [isLoading, setIsLoading] = useState<boolean>(true);
   const [error, setError] = useState<string>('');
   const [assetId, setAssetId] = useState('');
+  const [serialNumber, setSerialNumber] = useState('');
```

Comment:
Let's remove the `serialNumber` and use the `assetId` state because `asset_id` is a built-in query parameter for the `/sales` endpoint.